doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -351,9 +356,13 @@ The Infix command
 The :cmd:`Infix` command is a shortcut for declaring notations for infix
 symbols.
 
-.. cmd:: Infix @string := @one_term {? ( {+, @syntax_modifier } ) } {? : @scope_name }
+.. cmd:: Infix @notation_declaration
+
+   The command
+
+       :n:`Infix @string := @one_term {? ( {+, @syntax_modifier } ) } {? : @scope_name }`
 
-   This command is equivalent to
+   is equivalent to
 
        :n:`Notation "x @string y" := (@one_term x y) {? ( {+, @syntax_modifier } ) } {? : @scope_name }`
 
```

Comment:
To the risk of making the description longer (but more accurate), you could add "for x and y not occurring in @one_term", or 'for fresh x and y chosen such that it does not conflict with variables in @one_term", or something like that.

Reply:
Actually, the doc already says "where ``x`` and ``y`` are fresh names" (it's on the line below and it doesn't show in the diff).

Reply:
Ah, sorry for noise...